mne/defaults.py

```diff
@@ -15,19 +15,19 @@
     units=dict(mag='fT', grad='fT/cm', eeg='µV', eog='µV', ecg='µV', emg='µV',
                misc='AU', seeg='mV', dipole='nAm', gof='GOF', bio='µV',
                ecog='µV', hbo='µM', hbr='µM', ref_meg='fT', fnirs_raw='V',
-               fnirs_od='V', csd='V/m²'),
+               fnirs_od='V', csd='mV/m²'),
```

Comment:
@alexrockhill I think these should be correct now. Natively they are `V/m²`, but if we display them that way then `scalings['csd']` must be `1.`. Instead, if what we show in `evoked.plot` is `mV/m²`, then we need `scalings['csd'] = 1.e3`. So at least these are consistent now. They need to be related to one another. We should probably add a test for this, but it involves checking units and converting and would be a pain...

Reply:
SI unit test has been added that would have caught the unit problem in master